tutorials/vue/btnclick.md

```diff
@@ -0,0 +1,47 @@
+With Vue, you can tie button clicks to functions you want to execute. The formal
+way to do this is to use the `v-on` attribute, however, Vue has a neat shortcut, `@click`.
+
+
+```html
```

Comment:
Just put Vue code rather than the full HTML in this example. The HTML is just boilerplate.